Moonshot/Combat/bullet.tscn

```diff
@@ -18,6 +18,7 @@ animations = [ {
 } ]
 
 [node name="RigidBody2D" type="RigidBody2D"]
+collision_layer = 2147483664
```

Comment:
Are we sure we want to change the collision layer?

Reply:
This was because there was an issue with bullets colliding with themselves xD